Essentials/src/messages.properties

```diff
@@ -921,6 +921,7 @@ warpList={0}
 warpListPermission=\u00a74You do not have permission to list warps.
 warpNotExist=\u00a74That warp does not exist.
 warpOverwrite=\u00a74You cannot overwrite that warp.
+warpCreate=\u00a74You cannot create that warp.
```

Comment:
Maybe it would be better to call this `warpCanCreate` to be explicit, since `warpCreate` could be confused with `warpSet`.